### PR TITLE
[Notifier] [RocketChat] Add tests and pull up scheme check

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 5.2
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | ---
| License       | MIT
| Doc PR        | ---

### CODE REVIEW

**Review by derrabus — APPROVED** [review]

File: src/Symfony/Component/Notifier/Bridge/RocketChat/Tests/RocketChatTransportTest.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Notifier\Bridge\RocketChat\RocketChatTransport;
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+final class RocketChatTransportTest extends TestCase
+{
+    public function testToStringContainsProperties(): void
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Notifier/Bridge/RocketChat/Tests/RocketChatTransportTest.php]: `: void` :)

Comment (derrabus) [reply src/Symfony/Component/Notifier/Bridge/RocketChat/Tests/RocketChatTransportTest.php]: fabbot is not enough. We need a nicobot that searches PRs for `: void` in tests. 😄

Comment (OskarStark) [reply src/Symfony/Component/Notifier/Bridge/RocketChat/Tests/RocketChatTransportTest.php]: damn :-)

Comment (OskarStark) [reply src/Symfony/Component/Notifier/Bridge/RocketChat/Tests/RocketChatTransportTest.php]: Fixed

**Comment by OskarStark** [comment]
Ready to merge 👍

**Comment by OskarStark** [comment]
Wait, this must target `5.1` ❗

**Pull request closed without merging**